src/main/java/org/spongepowered/api/world/storage/WorldProperties.java

```diff
@@ -77,4 +79,12 @@
      * @return The difficulty
      */
     Difficulty difficulty();
+
+    /**
+     * Gets the {@link MutableWorldBorder}.
```

Comment:
Is it mutable? The return here seems to say it's just generic

Reply:
Hangover from the previous PR, I'll change it.